skymp5-server/cpp/papyrus_vm_lib/ActivePexInstance.cpp

```diff
@@ -142,7 +143,7 @@ VarValue CastToString(const VarValue& var)
       }
     }
     case VarValue::kType_Identifier:
-      assert(false);
+      throw std::runtime_error("Error");
       return VarValue();
```

Comment:
Here, something like that identifier type is not valid here. And the return statement makes no sense here anymore, so let's remove it